### DISCUSSION TITLE
Any plans for creating conan packages?

### DISCUSSION THREAD

**Discussion by handgunsoldier** [xmake-io/xmake#4732] [Q&A]
Xmake now supports importing conan packages very well, and thanks a lot!
Recently we come to a scenario that we want to publish our repos as conan packages, and our packages are built on `xmake.lua`. We know that the best practice is to migrate all our package management systems to `xrepo`. However, we are failed to persuade other partners to use it, for all kinds of reasons they insisted on. Thus our team continues to use `conan` to distribute and manage packages, and we need to package our repo to a conan center.
I found in `https://docs.conan.io/2/tutorial/creating_packages.html` that creating a conan package requires a `CMakeLists.txt` and a `conanfile.py`. The cmakelist is already supported to be generated. But I found the conanfile.py is somehow not ready to be dealt with.
Are there any plans to support the creation of conan packages?
@waruqi Thank you!

**Comment by waruqi** [comment]
I don’t know how to get conan to support it. Maybe I need to submit a PR to conan or submit a feature request to conan.
